x-pack/filebeat/input/gcs/client.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build !aix
+// +build !aix
```

Comment:
> What's your plan for automated testing of the GCS input?

@andrewkroh currently in the process of creating mocks and tests, will be done in a day or 2 .

Reply:
> What's the reason behind the `!aix` build tags?

I couldn't find proper docs showing the google cloud go sdk is supported on aix machines, hence to avoid compatibility issues skipped the aix build.

Reply:
Does the SDK state any of the platforms that is supports?

Does this compile for aix without CGO?

Reply:
I removed the !aix build tags and the ci has passed the cross compilation & build stages successfully, so seems like there are no issues. As for SDK docs listing platform supports, I couldn't find anything specific. A general os support for google cloud is listed in the docs but nothing specific to gcs storage sdks.

The ci-docs check & E2E tests for fleet are failing for some reason though.